### DISCUSSION TITLE
Turn off/disable when Windows screen times out

### DISCUSSION THREAD

**Discussion by ry-ol** [Drumber/RemoteLight#15] [Ideas]
Is it possible to have the software disable lights when Windows screen timeout occurs and then enable when the user wakes them back up? I've got my monitors set to 1 hour timeout and it'd be nice if the lights turned off as well. Could a Lua script achieve this?(I've never used Lua before)

**Comment by Drumber** [comment]
Hi, thanks for your suggestion. I have been thinking for a while to create a plugin, that executes custom defined actions on certain events. So I could implement a Screen-Timeout event or something like that. However, I currently have not much time and could not yet start working on the plugin.

You can also use the new REST API of RemoteLight v0.2.5 to toggle the active animation if you have some programming experience. HTTP requests are possible with almost every programming languages.
The REST API is documented [here](https://github.com/Drumber/RemoteLight/tree/master/remotelight-restapi) and beta builds for v0.2.5 can be downloaded from [here](https://gitlab.com/Drumber/RemoteLight/-/jobs/940709490/artifacts/browse/remotelight-client/target/).
